### PR TITLE
Logout button on profile page

### LINKED ISSUE #134: Logout button on profile page
On the profile page there should be a logout button that triggers a alert checking if the user wants to do this. Then it flushes the token and kicks the user to login page

### PR DESCRIPTION
**Pull request by adrian621**
#### What does this PR do?

Add logout button on profile screen

[134](#134 )

##### Why are we doing this? Any context or related work?

To be able to logout

#### Where should a reviewer start?

Profile.js

#### Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

closes #134

### CODE REVIEW

File: Bikeini/screens/Login.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ class Login extends React.Component {
   }
 
   componentDidMount() {
+    console.log('set state');
```
Comment (erikajansson1) [inline Bikeini/screens/Login.js]: Remove console.log

File: Bikeini/screens/Profile.js [hunk 2]
```diff
@@ -50,18 +52,24 @@ const styles = StyleSheet.create({
   },
   greenButton: {
     backgroundColor: '#44ccad',
-    left: '27%',
+    // left: '27%',
```
Comment (erikajansson1) [inline Bikeini/screens/Profile.js]: Remove

**Comment by adrianTyrevall** [comment]
Tagit bort console log

**Review by erikajansson1 — APPROVED** [review]
Looks good! :)

**Pull request merged**